scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3852,432 @@ def rvs(self, dim, size=1, random_state=None):
 
 
 unitary_group = unitary_group_gen()
+
+_mhg_doc_default_callparams = """\
+m : array_like
+    The number of different types of objects in
+    the population. That is, :math:`m[i]` is the
+    number of objects of type :math:`i`. 
+n : array_like
+    The number of samples taken from the population.
+"""
+
+_mhg_doc_callparams_note = \
+"""`m` must be an array of positive integers. Quantiles `x` must
+sum up to the sample size `n`. If the quantile :math:`i` contains
+values out of the range :math:`[0, m_i]` where :math:`m_i` is the
+number of objects of type :math:`i` in the population, appropriate
+values (like zero by ``pmf`` method) are returned by the called
+methods. If `m` or `n` contain negative values, the result will
+contain ``nan`` there.
+"""
+
+_mhg_doc_frozen_callparams = ""
+
+_mhg_doc_frozen_callparams_note = \
+    """See class definition for a detailed description of parameters."""
+
+mhg_docdict_params = {
+    '_doc_default_callparams': _mhg_doc_default_callparams,
+    '_doc_callparams_note': _mhg_doc_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+mhg_docdict_noparams = {
+    '_doc_default_callparams': _mhg_doc_frozen_callparams,
+    '_doc_callparams_note': _mhg_doc_frozen_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+
+class multivariate_hypergeom_gen(multi_rv_generic):
+    r"""
+    A multivariate hypergeometric random variable.
+
+    Methods
+    -------
+    ``pmf(x, m, n)``
+        Probability mass function.
+    ``logpmf(x, m, n)``
+        Log of the probability mass function.
+    ``rvs(m, n, size=1, random_state=None)``
+        Draw random samples from a multivariate hypergeometric
+        distribution.
+    ``mean(m, n)``
+        Mean of the multivariate hypergeometric distribution.
+    ``var(m, n)``
+        Variance of the multivariate hypergeometric distribution.
+    ``cov(m, n)``
+        Compute the covariance matrix of the multivariate
+        hypergeometric distribution.
+
+    Parameters
+    ----------
+    %(_doc_default_callparams)s
+    %(_doc_random_state)s
+
+    Notes
+    -----
+    %(_doc_callparams_note)s
+
+    Alternatively, the object may be called (as a function) to fix the `m`
+    and `n` parameters, returning a "frozen" multivariate hypergeometric
+    random variable:
+
+    The probability mass function for `multivariate_hypergeom` is
+
+    .. math::
+
+        P(X_1 = x_1, X_2 = x_2, \ldots, X_k = x_k) = \frac{\binom{m_1}{x_1}
+        \binom{m_2}{x_2} \cdots \binom{m_k}{x_k}}{\binom{M}{n}}, \\ \quad
+        (x_1, x_2, \ldots, x_k) \in \mathbb{N}^k \text{ with }
+        \sum_{i=1}^k x_i = n
+
+    where :math:`m_i` are the number of objects of type :math:`i`, :math:`M`
+    is the total number of objects in the population (sum of all
+    :math:`m_i`s), and :math:`n` is the size of the sample to be taken
```

Comment:
Not rendering properly.
![image](https://user-images.githubusercontent.com/6570539/93010588-101a6280-f543-11ea-9cea-8d1573b1723b.png)

```suggestion
    the :math:`m_i`), and :math:`n` is the size of the sample to be taken
```
Not sure if this will fix it because I'm guessing Sphinx doesn't like any character immediately adjacent to the backtick, but it's worth a shot.

Reply:
fixed